subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -102,6 +82,17 @@ config BT_FAST_PAIR_GATT_SERVICE
 
 if BT_FAST_PAIR_GATT_SERVICE
 
+config BT_FAST_PAIR_GATT_SERVICE_MODEL_ID
```

Comment:
why are we moving those configs?

Reply:
So that Fast Pair Kconfig are more clearly separated from Bluetooth ones (comment banner)

this is just a minor improvement